Makefile

```diff
@@ -12,6 +12,7 @@ endif
 VERSION = $(shell git describe --dirty --tags --always)
 GIT_COMMIT = $(shell git rev-parse HEAD)
 K8S_VERSION = v1.18.2
+OLM_VERSION = 0.15.1
```

Comment:
It is not passing in the sanity-test. See: https://travis-ci.org/github/operator-framework/operator-sdk/jobs/706044223#L950

Reply:
I'm hesitant to include the OLM version in an SDK release. Thoughts @joelanford @shawn-hurley?

Reply:
The Assets file generated by bin-data is not passing the CI. That file needs to be whitelisted. Will try doing that.

Reply:
HI @varshaprasad96 the CI error is: 

```
Checking format of error and log messages...
Error messages do not begin with lower case:
/home/travis/gopath/src/github.com/operator-framework/operator-sdk/internal/olm/olm-bindata-0.15.1.go:130:			return nil, fmt.Errorf("Asset %s can't read by error: %v", name, err)
/home/travis/gopath/src/github.com/operator-framework/operator-sdk/internal/olm/olm-bindata-0.15.1.go:134:	return nil, fmt.Errorf("Asset %s not found", name)
/home/travis/gopath/src/github.com/operator-framework/operator-sdk/internal/olm/olm-bindata-0.15.1.go:156:			return nil, fmt.Errorf("AssetInfo %s can't read by error: %v", name, err)
/home/travis/gopath/src/github.com/operator-framework/operator-sdk/internal/olm/olm-bindata-0.15.1.go:160:	return nil, fmt.Errorf("AssetInfo %s not found", name)
/home/travis/gopath/src/github.com/operator-framework/operator-sdk/internal/olm/olm-bindata-0.15.1.go:199:				return nil, fmt.Errorf("Asset %s not found", name)
/home/travis/gopath/src/github.com/operator-framework/operator-sdk/internal/olm/olm-bindata-0.15.1.go:204:		return nil, fmt.Errorf("Asset %s not found", name)
Makefile:249: recipe for target 'test-sanity' failed
make: *** [test-sanity] Error 255 
```

> `The Assets file generated by go-bindata is not passing the CI. That file needs to be whitelisted.`
it is not the root cause of the issue. see the error msg `Error messages do not begin with lower case:`

Reply:
Hi @camilamacedo86. The entire `olm-bindata-0.15.1.go` file is generated using `go bindata` which shouldn't be modified. Hence, that needs to be whitelisted.

Reply:
Hi @varshaprasad96,  

It is NOT a good practice to commit something that is built. We are getting the OLM manifests and building the bindata to ship it inside of the SDK binary. It is not useful outside of it and we should not commit the bindata in the same way that we do not commit the SDK binary. 

If we cannot relay on OLM and expected they can remove the manifests that are used to gen the bindata then we should copy and commit in the repo the manifest files and NOT the bindata generated with them. At the same way that we do not commit the SDK binary and just the code that used to gen it. 

See for example the Kubernetes repo working with https://github.com/kubernetes/kubernetes/search?q=bindata&unscoped_q=bindata

They are using the build dir or the test dir to output the bindata when it is required but the file is never committed for the same reasons. 

c/c @jmrodri

Reply:
Hi @varshaprasad96, 

IMO we should never commit the built file. See: https://github.com/operator-framework/operator-sdk/pull/3297/files#r453211330

Reply:
I think we agreed in commit to solving the problem,